docs/core-concepts/run-udfs/multiple_jobs.mdx

```diff
@@ -0,0 +1,212 @@
+Using the same UDF as above we can run this UDF 10 times on an "offline" instance:
+
+```python showLineNumbers
+# We'll run this UDF 10 times
+job = udf(arg_list=range(10))
+job.run_remote()
+```
+
+This pushes a job onto a persistent machine on Fused server.
+
+<details>
+  <summary>`run_remote` additional arguments</summary>
+
+    With `job.run_remote()` you also have access to a few other arguments to make your remote job fit your need:
+
+    - `instance_type`: Decide which type of machine to run your job on (based on the [AWS General Purpose instance types](https://aws.amazon.com/ec2/instance-types/))
+    - `disk_size_gb`: The amount of disk space in Gb your instance requires
+
+    For example if you want a job with 16 vCPUs, 64Gb of RAM and 100Gb of storage you can call:
+    ```python showLineNumbers
+    job.run_remote(instance_type="m5.4xlarge", disk_size_gb=100)
+    ```
+
+</details>
+
+{/* This is missing a section on how to get the output from the job directly, similar to runner.get_concat() in the other section */}
+
+#### Accessing "offline" job logs
+
+You can view the logs of all your on-going and past `run_remote()` jobs either:
+
+import ReactPlayer from 'react-player';
+import ImgRunRemoteLogs from '@site/docs/core-concepts/run-udfs/run_remote_output.png';
+import ImgRunRemoteEmail from '@site/docs/core-concepts/run-udfs/run-remote-email.png';
+
+<details>
+  <summary>In a notebook</summary>
+
+    Running `job.run_remote()` in a notebook gives you a clickable link:
+
+    <div style={{textAlign: 'center'}}>
+    <img src={ImgRunRemoteLogs} alt="Dark Vessel Detection workflow" style={{width: 800}} />
+    </div>
+
+</details>
+
+<details>
+  <summary>In Fused Workbench</summary>
+
+    Under the "Jobs" tab, on the bottom left of Workbench:
+
+    <ReactPlayer className="video__player" playing={false} muted={true} controls height="100%" url="https://fused-magic.s3.us-west-2.amazonaws.com/workbench-walkthrough-videos/docs_rewrite/core-concepts/run-udfs/run-remote-job-logs.mp4" width="100%" />
+
+</details>
+
+<details>
+  <summary>By email (you'll receive 1 email for each job)</summary>
+
+    Each job leads to an email summary with logs upon completion:
+
+    <div style={{textAlign: 'center'}}>
+    <img src={ImgRunRemoteEmail} alt="Dark Vessel Detection workflow" style={{width: 800}} />
+    </div>
+
+</details>
+
+
+#### Getting "offline" results
+
+To get data back from your "offline" run is a bit more complicated than for ["real-time"](/core-concepts/run-udfs/run_multiple/#getting-real-time-results). Our recommendation is to have your UDF write data directly to disk or cloud storage and access it after
+
+<details>
+  <summary>Example job: saving to disk</summary>
+
+    A common use case for offline jobs is as a "pre-ingestion" process. You can find a real-life example of this in our [dark vessel detection example](/user-guide/examples/dark-vessel-detection/#32---writing-a-udf-to-open-each-ais-dataset)
```

Comment:
This is based on my experience, I don't know if we want to recommend users do something else?